network-api/networkapi/wagtailpages/rss.py

```diff
@@ -32,25 +44,21 @@ def items(self):
         feed_set = cache.get("rss_feed_set")
 
         if feed_set is None:
-            # If we don't have an active cache, we build one: pull the index page
-            # specifically using the English page title, as an IndexPage rather than
-            # as a BlogIndexPage, to make sure we're not filtering out all the
-            # "featured" posts (which we need to do for site content purposes).
+            # If we don't have an active cache, we build one: First, retrieve the BlogIndexPage.
             try:
-                index = IndexPage.objects.get(title__iexact="Blog", locale=request_locale)
+                blog_index = self.get_blog_index_page()
 
             except IndexPage.DoesNotExist:
-                # If that doesn't yield the blog page, pull using the universal title
-                try:
-                    index = IndexPage.objects.get(title__iexact="Blog", locale=request_locale)
+                # If that doesn't yield the blog index page, there's not much we can do other than to pretend
+                # there are no posts to serialize to RSS/Atom format.
+                return []
 
-                except IndexPage.DoesNotExist:
-                    # At this point there's not much we can do other than to pretend
-                    # there are no posts to serialize to RSS/Atom format.
-                    return []
```

Comment:
Hi @jhonatan-lopes just wanted to call out that I removed this second try/except block because it seemed redundant 👍